### ISSUE TITLE
Reflection artifacts when source turned off

### ISSUE THREAD

**Issue by arouinfar** [phetsims/wave-interference#38]
For #31 

In Unfuddle Ticket [#2535](https://phet.unfuddle.com/a#/projects/9404/tickets/by_number/2535), Wendy Adams reported:
> Also after you turn faucets off for a bit the surface of the water keeps vibrating.  (I’ve turned my tank and have the graph open).  After turning the water off I can see a reflection come back after a bit. If you let it sit for a long time, the vibration appears to build up.  I don’t think it’s supposed to do this but can’t remember for sure.  It might have been something we decided to live with.  Just wanted to point it out.
The same thing happens in the sound tab.  A few reflections are seen and then it settles into a continuous vibration. The particle view however appears perfectly stationary, just the graph vibrates.
If you drag the barrier back and forth  a bit, you can get huge waves that keep vibrating nice saw tooth waves that never settle down.  Not very physical.

Assigning to @samreid for investigation.

**Comment by samreid** [comment]
> If you drag the barrier back and forth a bit, you can get huge waves that keep vibrating nice saw tooth waves that never settle down.

This has been addressed by using a new plane wave model.  There are still other artifacts from the boundary reflections.  At one meeting we discussed the possibility of increasing damping everywhere in the lattice to mitigate the problem, but decided against it.  We also argued that some reflections seemed reasonable, if we are mimicking a wave pool.

@arouinfar does that sound accurate to you?  Anything else to do for this issue?
